icubam/backoffice/handlers/upload.py

```diff
@@ -28,13 +28,13 @@ def post(self) -> None:
       return self.answer(f'No CSV content', error=True)
 
     sync = synchronizer.CSVSynchronizer(self.db)
-    sync_fns: Dict[str, Callable[..., int]] = {
-      'user': sync.sync_users_from_csv,
-      'icu': sync.sync_icus_from_csv,
-      'bedcounts': sync.sync_bedcounts_from_csv
+    sync_fns: Dict[int, Callable[..., int]] = {
+      base.ObjType.USERS.value: sync.sync_users_from_csv,
+      base.ObjType.ICUS.value: sync.sync_icus_from_csv,
+      base.ObjType.BEDCOUNTS.value: sync.sync_bedcounts_from_csv
     }
     objtype = data.get('objtype', None)
-    sync_fn = sync_fns.get(objtype, None)
+    sync_fn = sync_fns.get(int(objtype), None)
```

Comment:
If we pass the enum name in the API, I think here we could do, 
```suggestion
    objtype_name = data.get('objtype', None)
    objtype = base.ObjType[objtype_name]
    sync_fn = sync_fns.get(objtype, None)
```
and use `base.ObjType.USERS` etc. as hash keys directly above.